tests/unit/subscriptions/test-class-wc-payments-subscription-change-payment-method.php

```diff
@@ -0,0 +1,192 @@
+<?php
+/**
+ * Class WC_Payments_Invoice_Service_Test
+ *
+ * @package WooCommerce\Payments\Tests
+ */
+
+use PHPUnit\Framework\MockObject\MockObject;
+use WCPay\Exceptions\API_Exception;
+
+/**
+ * WC_Payments_Invoice_Service_Test unit tests.
+ */
+class WC_Payments_Subscription_Change_Payment_Method_Test extends WP_UnitTestCase {
+
+	/**
+	 * System under test.
+	 *
+	 * @var WC_Payments_Product_Service
+	 */
+	private $change_payment_method_handler;
+
+	/**
+	 * Pre-test setup
+	 */
+	public function setUp() {
+		parent::setUp();
+
+		$this->change_payment_method_handler = new WC_Payments_Subscription_Change_Payment_Method_Handler();
+	}
+
+	/**
+	 * Tests for WC_Payments_Subscription_Change_Payment_Method_Handler::update_subscription_change_payment_button().
+	 */
+	public function test_update_subscription_change_payment_button() {
+		$mock_subscription = new WC_Subscription();
+
+		// Test the false case - no change to input.
+		$this->assertEquals( [], $this->change_payment_method_handler->update_subscription_change_payment_button( [], $mock_subscription ) );
```

Comment:
Adding a few assertion nitpicks, feel free to ignore them if they're not helpful 😄

`assertSame` is typically a little nicer than `assertEquals`

Reference: paJDYF-17V-p2

Reply:
TIL thanks @aprea!